Airports/EGWU/SMR/Regions.txt

```diff
@@ -0,0 +1,562 @@
+; - Background
```

Comment:
```suggestion
;Background
```
If you could go through this whole file and find & replace "; - " with ";" that would be great!

Reply:
I have made these changes for all the pending SMR's